program/src/processor.rs

```diff
@@ -4476,7 +4525,10 @@ impl Processor {
 
         let mut mango_account =
             MangoAccount::load_mut_checked(mango_account_ai, program_id, mango_group_ai.key)?;
-        check!(&mango_account.owner == owner_ai.key, MangoErrorCode::InvalidOwner)?;
+        check!(
+            &mango_account.owner == owner_ai.key || &mango_account.delegate == owner_ai.key,
+            MangoErrorCode::InvalidOwner
+        )?;
```

Comment:
Sure about this one? withdraw_msrm sounds withdraw-y, should the delegate be allowed to do this?

Reply:
thanks, correct, fixed